### PR TITLE
feat(art-boards:vue-chat-app): added new vue chat app to art boards e…

### PR DESCRIPTION
**Pull request by AlexTraher**
First pass at adding the vue chat app to art-boards example.

While this functionality is working, I've marked it as a draft as there are some issues with the TS setup conflicting between vue and React jsx when the `VueChatApp` is imported.

### CODE REVIEW

**Comment by alexlbr** [comment]
This PR looks good, good job! 

I think that TS issue is tricky to solve. Based on [this comment](https://github.com/johnsoncodehk/volar/discussions/592#discussioncomment-2163181), we could try the following:
```ts
// ./app/monolith/stubs/chat-app-vue.d.ts
export default GetComposableAppAsync;
```

Then in  .app/monolith/tsconfig.json:
```json
{
  "extends": "../../tsconfig.json",
  "include": ["."],
  "exclude": ["dist", "build", "node_modules"],
  "compilerOptions": {
    "paths": {
      "@art-boards/chat-app-vue": ["./stubs/chat-app-vue.d.ts"] 👈
    }
  }
}

```

Regarding the [CI issue](https://github.com/leanjs/leanjs/actions/runs/3554918588/jobs/5971321799#step:5:25), I think you need to add `@leanjs/webpack-vue` to the [package.json](https://github.com/leanjs/leanjs/blob/main/examples/art-boards/package.json#L20) of the art-boards repo to fix it. The art-boards is a repo inside another repo, it's not a workspace. The [CI only installs dependencies in the art-boards directory](https://github.com/leanjs/leanjs/blob/main/.github/workflows/example_pr_art_boards.yml#L12). 

Other than that, it's ready to be merged 🚀

File: examples/art-boards/composable-apps/chat-vue/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "lint": "tsc --noEmit",
+    "deploy:pr": "lean deploy"
+  },
+  "main": "./src/index.ts",
+  "author": "",
+  "license": "MIT",
+  "peerDependencies": {
+    "@leanjs/core": "*",
+    "@leanjs/vue": "*",
+    "vue": "*"
```
Comment (alexlbr) [inline examples/art-boards/composable-apps/chat-vue/package.json:17]: We also need `vue` as a `dependency` in the root package.json

Comment (AlexTraher) [reply examples/art-boards/composable-apps/chat-vue/package.json:17]: Yep - done that now :+1:

**Comment by AlexTraher** [comment]
That TS suggestion did the trick - nice find! I've made all the changes you've suggested, but I'm not quite sure why the build hasn't triggered for my latest commit?

**Review by alexlbr — APPROVED** [review]

**Pull request merged**